README.md

```diff
@@ -1,78 +1,78 @@
-
-   - GET `/api/events/:city/:stateCode/:startDateTime?/:endDateTime?`
-   - Parameters:
-     - `city` (required): Name of the city
-     - `stateCode` (required): Two-letter state code
-     - `startDateTime` (optional): Start date in YYYY-MM-DD format
-     - `endDateTime` (optional): End date in YYYY-MM-DD format
-
-   Example requests:
-
-   - `/api/events/Seattle/WA/2024-02-01/2024-02-05`
-   - `/api/events/Seattle/WA`
-   - `/api/events/Seattle/WA//2024-02-01`
-
-## Setup Instructions
-
-1. Create a folder to contain both the front-end and back-end repos
-2. Clone this repository to that folder
-3. Run `npm install` to install dependencies
-4. Create a `.env` file and add your Ticketmaster API key:
+# Back-End Repo for Node/React Practicum
+
+This will be the API for the front-end React app part of your practicum project.
+
+These instructions are for the **front-end team** so they can setup their local development environment to run
+both the back-end server and their front-end app. You can go through these steps during your first group meeting
+in case you need assistance from your mentors.
+
+> The back-end server will be running on port 8000. The front-end app will be running on port 3000. You will need to run both the back-end server and the front-end app at the same time to test your app.
+
+### Setting up local development environment
+
+1. Create a folder to contain both the front-end and back-end repos
+2. Clone this repository to that folder
+3. Run `npm install` to install dependencies
+4. Pull the latest version of the `main` branch (when needed)
+5. Run `npm run dev` to start the development server
+6. Open http://localhost:8000/api/v1/ with your browser to test.
+7. Your back-end server is now running. You can now run the front-end app.
+
+#### Running the back-end server in Visual Studio Code
+
+Note: In the below example, the group's front-end repository was named `bb-practicum-team1-front` and the back-end repository was named `bb-practicum-team-1-back`. Your repository will have a different name, but the rest should look the same.
+![vsc running](images/back-end-running-vsc.png)
+
+#### Testing the back-end server API in the browser
+
+![browser server](images/back-end-running-browser.png)
+
+> Update the .node-version file to match the version of Node.js the **team** is using. This is used by Render.com to [deploy the app](https://render.com/docs/node-version).
+
+-
+-
+-
+-
+-
+-
+-
+-
+
+# Back-End Repo for Node/React Practicum
+
+This is the back-end API server for the front-end React app. It provides endpoints for event data from Ticketmaster API, user data and authentication, as well as saved user interests and itineraries from our MongoDB database.
+
+## API Endpoints
+
+### Base URL
+
+`http://localhost:8000/api/v1`
+
+### Endpoints
+
+1. **Main endpoint**
+
+   - GET `/`
+   - Returns a simple test message
+
+2. **Ticketmaster Events**
+
+   - GET `/api/events/:city/:stateCode/:startDateTime?/:endDateTime?`

```

Comment:
I just realized this endpoint needs to be edited as its not accurate, but I can fix that on the parent branch.